includes/options/class-amp-options-manager.php

```diff
@@ -603,7 +603,7 @@ public static function render_stories_deprecation_notice() {
 		} elseif ( ! self::is_stories_experience_enabled() && 'toplevel_page_' . self::OPTION_NAME === get_current_screen()->id ) {
 			printf(
 				'<div class="notice notice-info"><p>%s</p></div>',
-				esc_html__( 'The Stories experience has been removed from the AMP plugin. This beta feature is being split into a separate standalone plugin which will be available for installation soon.', 'amp' )
+				esc_html__( 'The Stories experience [beta] has been split into a standalone plugin which will be available soon.', 'amp' )
```

Comment:
There is one more instance of this string in the `render_stories_deprecation_editor_notice()` method.

Reply:
Or rather, perhaps two?

https://github.com/ampproject/amp-wp/blob/e42d45a4863a2afeaee2f723d61c5d9e16f93d6f/includes/options/class-amp-options-manager.php#L596

https://github.com/ampproject/amp-wp/blob/e42d45a4863a2afeaee2f723d61c5d9e16f93d6f/includes/options/class-amp-options-manager.php#L631

Reply:
I would strongly advocate for removing the `[beta]` part from this string.

Also, I would unify the strings. Now there are "has been split" and "is being split"

Reply:
+1